### PR TITLE
Document outlining project deliverables

### PR DESCRIPTION
**Pull request by mbonsma**
We can upload this as a pdf on Quercus. It contains:

* A description of the project and project goals
* Guidelines for choosing their own data if they want to
* Detailed project deliverables
* Grading rubric

We still need to add the link to the datasets, I'm waiting on that because we'll be updating the name of the datasets lecture.

### CODE REVIEW

File: project_info.Rmd [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+### Data
+
+A list of recommended datasets can be found [here](). You are welcome to choose a
+dataset not listed, or data collected as part of a research project, but keep in mind
+that you may not submit anything twice: any work you do as part of this course may not
+be submitted for credit in another course (such as a fourth-year research project) and vice versa. 
+If choosing a dataset not listed, make sure it is well-documented, legitimate, and 
+complex enough to support your analysis efforts. Your work should be original; your
+project should not be a reproduction of published analysis.
```
Comment (James-S-Santangelo) [inline project_info.Rmd]: Change 'analysis' to 'analyses'

File: project_info.Rmd [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+encouraged to publish their work on [figshare](https://figshare.com/), an open, 
+citable repository of scientific content. 
+
+All items (except the presentation) are due on December 5th at 11:59 pm.
+
+## Project submission
+
+The project report and code should be submitted on GitHub - each group will have 
+their own GitHub repository to which you can upload your report and code. 
+
```
Comment (James-S-Santangelo) [inline project_info.Rmd:70]: Should we mention something about the format for the final report? For example, should they have Rmd and pdf files on their GitHub or will we accept word documents?

File: project_info.Rmd (hunk 1 above)
Comment (mbonsma) [reply project_info.Rmd]:
```suggestion
project should not be a reproduction of published analyses. 
```

File: project_info.Rmd (hunk 2 above)
Comment (mbonsma) [reply project_info.Rmd:70]: That's something I want to discuss at our meeting - we want it to be reproducible, so should we make them write the whole report as an R markdown document? Then they would submit pdf or html plus .Rmd.

**Pull request merged**